android/build.gradle

```diff
@@ -15,13 +15,11 @@ buildscript {
     }
 
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
         classpath "org.jetbrains.kotlin:kotlin-serialization:$kotlin_version"
-        classpath 'com.android.tools.build:gradle:8.5.0'
+        classpath 'com.android.tools.build:gradle:8.7.0-beta01'
```

Comment:
@atavism Which Android studio version you are using? this is connected to the AS version Unless we get an incompatible error. Also why we are using beta version and not stable one?

Reply:
The beta one is the only one I could get working with Android Studio. Are you using the stable version?

Reply:
Could you please try [Android Studio Jellyfish | 2023.3.1](https://developer.android.com/studio/archive) and keep gradle version same as 8.5.0

Reply:
Yes doing that now

Reply:
I'm happy to downgrade Android Studio, but just curious, does the latest version in the stable channel (Koala Feature Drop) not work for you? That's the version I've been using and it supports AGP 8.5-8.6

![Screenshot 2024-09-16 at 4 27 47 AM](https://github.com/user-attachments/assets/d833e246-70e9-4908-a08f-ff27d45d0559)

Reply:
Sure, I can upgrade to Koala,